packages/xo-web/src/common/intl/messages.js

```diff
@@ -833,6 +833,9 @@ const messages = {
   addHostNotHomogeneousErrorMessage: 'Host patches could not be homogenized.',
   disconnectServer: 'Disconnect',
 
+  // ----- Host item ------
+  noMoreMaintained: 'This current host version is no more maintained.',
```

Comment:
```suggestion
  noMoreMaintained: 'This host version is no more maintained.',
```

Reply:
The correct wording for this that is grammatically correct would be the following;

This host version is no longer maintained, please upgrade to _supportedversion_

Reply:
Or you can have the following as well;

This host version is no longer maintained.

Or

This host version is no longer maintained, please upgrade to _supportedversion_

Reply:
Let's go with "This host version is no longer maintained". Thanks @L1ghtn1ng!